### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by mshitrit**
as part of [ECOPROJECT-1662](https://issues.redhat.com//browse/ECOPROJECT-1662) Adding events for better visibility of SNR remediation process.

### CODE REVIEW

**Review by clobrano — COMMENTED** [review]
I left just a note, otherwise it's ok for me

File: controllers/selfnoderemediation_controller.go [hunk 1]
```diff
@@ -394,6 +415,7 @@ func (r *SelfNodeRemediationReconciler) remediateWithResourceRemoval(snr *v1alph
 			if err = r.updateConditions(remediationFinishedNodeNotFound, snr); err != nil {
 				return ctrl.Result{}, err
 			}
+			r.Recorder.Event(snr, eventTypeNormal, eventReasonRemediationStopped, "couldn't find node matching remediation")
```
[the remark this answers is not in the document]
Comment (mshitrit) [reply controllers/selfnoderemediation_controller.go:418]: TBH not sure myself, no strong opinion either way.
Log was info (can be either Info or Error) so I followed suite with the event.

**Comment by clobrano** [comment]
/lgtm 
Giving others a chance to review as well, feel free to unhold
/hold

**Pull request merged**